src/utils/saml.ts

```diff
@@ -0,0 +1,21 @@
+import { DOMParser } from "xmldom";
+
+export interface ISamlResponse {
+  RelayState: string;
+  SAMLResponse: string;
+}
+
+export const getSamlIssuer = (body: ISamlResponse): string | undefined => {
+  const decodedResponse =
+    body && body.SAMLResponse
+      ? Buffer.from(body.SAMLResponse, "base64").toString("utf8")
+      : "";
+  const domParser = new DOMParser().parseFromString(decodedResponse);
+  if (domParser === undefined) {
+    return undefined;
+  }
+  const samlIssuerTAG = "saml:Issuer";
+  const samlIssuerElements = domParser.getElementsByTagName(samlIssuerTAG);
+  const issuers = Array.from(samlIssuerElements).map(_ => _.textContent);
```

Comment:
why you need `Array.from` ?

Reply:
Because `getElementsByTagName()` return an `HTMLCollectionOf<Element>` that is an `Iterable` even if no one or only one element is present.

Reply:
I could use `item(0)`. I'll fix it

Reply:
fixed in d098c6c